docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -389,18 +173,25 @@ Configurations, Producer PropertiesProducer Properties, and Advanced Producer Co
 |LoadBalancingStrategy
 |The load balancing strategy.
 
+|*spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer*.load-balancing.partition-ownership-expiration-interval
+|Duration
+|The time duration after which the ownership of partition expires.
+
 |*spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer*.track-last-enqueued-event-properties
 |Boolean
-| Whether the event processor should request information on the last enqueued event on its associated partition, and track that information as events are received.
+|Whether the event processor should request information on the last enqueued event on its associated partition, and track that information as events are received.
 
-|*spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer*.partition-ownership-expiration-interval
-|Boolean
-| The expiration interval time for partition ownership
+|*spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer*.prefetch-count
+|Integer
+|The count used by the consumer to control the number of events the Event Hub consumer will actively receive and queue locally.
+
+|*spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer*.initial-partition-event-position
+|Map of `StartPositionProperties`
+|The map containing the event position to use for each partition if a checkpoint for the partition does not exist in checkpoint store. This map is keyed off of the partition id.
 |===
 
 ====== Advanced Consumer Configuration
-The configuration in the above first 2 sections(`Azure Common Configuration Options`, `Azure
-Event Hubs Client Configuration Options`) can be applied for each specific consumer by replacing the prefix of *spring.cloud.azure.eventhubs* with *spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer*.
+The above <<eventhubs-connection-configration, connection>>, <<Checkpoint Configuration Properties, checkpoint>> and <<Configuration.adoc#configuration, common Azure SDK client>> configuration are supported to be customized for each binder consumer, which can be configured with the prefix *spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer.*.
```

Comment:
*spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer.* to `spring.cloud.stream.eventhubs.bindings.<binding-name>.consumer.`